rotimi-best/classroomio
# Pranay911/issue156

## What does this PR do?

This PR adds a new feature titled "Hover on Navigation bar" to the project. The feature enhances user experience by enabling hover functionality on the navigation bar. It improves navigation intuitiveness and accessibility for users.



Fixes #156 

![2](https://github.com/rotimi-best/classroomio/assets/65008142/525ec001-e9e2-4401-af1d-d495e32f1fe6)

## Type of change




## How should this be tested?

1. Navigate to the navigation bar.
2. Hover over each navigation item.
3. Verify that a visual indicator (e.g., color change, underline) appears upon hover.
4. Ensure that the navigation functionality remains intact.
5. Test the feature on different devices and browsers to ensure compatibility.

### Required
